packages/datetime/src/dateInput.tsx

```diff
@@ -488,19 +493,23 @@ export class DateInput extends AbstractComponent<IDateInputProps, IDateInputStat
             const tabbableElements = this.contentRef.querySelectorAll("input, [tabindex]:not([tabindex='-1'])");
             const numOfElements = tabbableElements.length;
             if (numOfElements > 0) {
-                const lastPopoverElement = tabbableElements[numOfElements - 1] as HTMLElement;
-                if (this.lastPopoverElement !== lastPopoverElement) {
+                // Keep track of the last focusable element in popover and add
+                // a keydown handler, so that:
+                // * popover closes when the user tabs to the next element
+                // * or focus moves to previous element if shift+tab
```

Comment:
something is wrong with the language in this comment. it doesn't parse correctly. 

i think "popover closes:" belongs after "so that", but then the second bullet sounds incorrect.